### DISCUSSION TITLE
More Information / Roadmap / What is yeti feeds frontend?

### DISCUSSION THREAD

**Discussion by ghost** [yeti-platform/yeti#1085] [General]
Hello,

this looks like a really cool project. Though it would be great if there were more information and a roadmap.
While looking at the repos i found the repo yeti-feeds-frontend. Is this a replacement of the current ui or something else?

**Comment by tomchop** [comment]
Hi, and sorry for the very long dealy in answering! We don't currently have an official roadmap, besides the open issues in both projects.

As to yeti-feeds-frontend, it's actually the frontend part of yeti (the yeti repo doesn't have any UI/frontend code itself).

Both repos are used to generate the docker images that we offer for production deployment :)
